### PR TITLE
Update gh-collaboration.qmd

### PR DESCRIPTION
**Pull request by n-equals-one**
Added new branch section

### CODE REVIEW

File: github/gh-collaboration.qmd [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # How to collaborate on Github
 
 ## Making a branch
+- On the main page of the repository click on the branch dropdown menue.
+- View all branches
+- Click New branch, give it a name and select the branch source.
+- Click create branch
```
Comment (MartinPippel) [inline github/gh-collaboration.qmd:7]: all fine, just wanted to test a block comment

**Review by MartinPippel — APPROVED** [review]
all fine

File: github/gh-collaboration.qmd [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # How to collaborate on Github
 
 ## Making a branch
+- On the main page of the repository click on the branch dropdown menue.
```
Comment (mahesh-panchal) [inline github/gh-collaboration.qmd:4]:
```suggestion
- On the main page of the repository click on the branch dropdown menu.
```
Perhaps add the location of the drop-down menu on the page, i.e describe where it's placed.

I just realised that perhaps you're not referring to the drop-down menu but the button branches next to it. In which case you don't need 
```
- View all branches
```

Then then clears up what you're referring to in the sentence beneath the bullet points.

Comment (n-equals-one) [reply github/gh-collaboration.qmd]: I guess my point was that you  can either directly quickly select a branch OR go via "view all branches", meaning that there are more than one way to achieve the same thing,  but I see your point.

Comment (mahesh-panchal) [reply github/gh-collaboration.qmd]: Can you clarify this please in the text, as I didn't find the distinction clear.

File: github/gh-collaboration.qmd [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # How to collaborate on Github
 
 ## Making a branch
+- On the main page of the repository click on the branch dropdown menu.
+- View all branches
+- Click New branch, give it a name and select the branch source.
+- Click create branch
+
+You can also make a branch by clicking on the dropdown menue and give an unique name in the "Find or create branch..." field followed by clicking Create branch.
```
Comment (mahesh-panchal) [inline github/gh-collaboration.qmd:9]:
```suggestion
You can also make a branch by clicking on the drop-down menu and give an unique name in the "Find or create branch..." field followed by clicking Create branch.

```

File: github/gh-collaboration.qmd (hunk 2 above)
Comment (n-equals-one) [reply github/gh-collaboration.qmd]: OK, updated to clarify this a bit better.

File: github/gh-collaboration.qmd [hunk 4]
```diff
@@ -1,6 +1,13 @@
 # How to collaborate on Github
 
 ## Making a branch
+- On the main page of the repository go to the file tree view on the left and click on the branch dropdown menu.
```
Comment (mahesh-panchal) [inline github/gh-collaboration.qmd:4]: Sorry, what's the "file tree view"? I don't mean to be a pain on this, but following this, I'm not sure what you're referring to. The text should be unambiguous to a novice.

Comment (n-equals-one) [reply github/gh-collaboration.qmd:4]: Hmm, I took that exact formulation from here (the official documentation): 
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository

I will make up my own novice word for it ;-), thanks for pointing it out.

Comment (mahesh-panchal) [reply github/gh-collaboration.qmd:4]: I guess the main difference is the use of pictures and icons there. We can try and enable the fontawesome icons if you think that will help.

Comment (mahesh-panchal) [reply github/gh-collaboration.qmd:4]: I think embedding their picture should be fine too as long as you attribute it.

Comment (mahesh-panchal) [reply github/gh-collaboration.qmd:4]: I have to admit though it feels a bit long-winded given the `branches` button to the right of it, that brings one to the same page in less clicks.

Comment (n-equals-one) [reply github/gh-collaboration.qmd:4]: I actually added one more sentence and the link. Hopefully it is clear now.

**Review by mahesh-panchal — APPROVED** [review]

**Pull request merged**